packages/runtime/test/integration/basic-teal.ts

```diff
@@ -42,9 +42,13 @@ describe("Algorand Smart Contracts", function () {
     // check initial balance
     assert.equal(john.balance(), initialJohnHolding);
     assert.equal(bob.balance(), initialBobHolding);
+    // get logic signature
+    const lsig = runtime.getLogicSig(getProgram('basic.teal'), []);
+    lsig.sign(john.account.sk);
```

Comment:
Please update the title to reflect stateless ASC `delegated signature mode`.

Reply:
ok